lib/errors.js

```diff
@@ -149,6 +149,14 @@ const codes = {
     'FST_ERR_DEC_REFERENCE_TYPE',
     "The decorator '%s' of type '%s' is a reference type. Use the { getter, setter } interface instead."
   ),
+  FST_ERR_DEC_UNDECLARED: createError(
+    'FST_ERR_DEC_UNDECLARED',
+    "No decorator '%s' has been declared."
+  ),
+  FST_ERR_DEC_GET_ACCESS_AFTER_START: createError(
+    'FST_ERR_DEC_GET_ACCESS_AFTER_START',
+    'Methods getDecorators should be called before the application start.'
```

Comment:
```suggestion
    'Method getDecorators should be called before the application start.'
```

Ooops, the idea here is to encourage to use this feature at boot time, there is not interest using later I think.